sound/soc/sof/ipc4-topology.c

```diff
@@ -675,6 +677,7 @@ static int sof_ipc4_widget_setup_comp_pipeline(struct snd_sof_widget *swidget)
 	pipeline->msg.primary |= SOF_IPC4_MSG_TARGET(SOF_IPC4_FW_GEN_MSG);
 
 	pipeline->msg.extension = pipeline->lp_mode;
+	pipeline->msg.extension |= SOF_IPC4_GLB_PIPE_EXT_CORE_ID(pipeline->core_id);
```

Comment:
I still don't understand what happens if the pipeline core_id is not aligned with the core_id used for the components of said pipeline, e.g. for DP processing.

Reply:
I will update with some protection, I just need to find the correct place.